docs/source/api/backend/index.rst

```diff
@@ -22,12 +22,44 @@ Available backends
         :img-background: ../../_static/bokeh-logo-light.svg
         :class-img-bottom: dark-light
 
+    .. grid-item-card::
+        :link: plotly
+        :link-type: doc
+        :link-alt: plotly
+        :img-background: ../../_static/plotly-logo-light.png
+        :class-img-bottom: dark-light
+
+    .. grid-item-card::
+        :link: none
+        :link-type: doc
+        :link-alt: none
+
+        .. grid:: 3
+
+           .. grid-item::
+              
+              .. image:: ../../_static/no-image.svg
+                 :class: dark-light
+                 :alt:
+
+           .. grid-item::
+
+              Data
+
+           .. grid-item::
+
+              .. raw:: html
+
+                 <s>Viz</s>
```

Comment:
I think the best way forward would be to combine everything into an image. But didn't want to get into that to prematurely. I am still not sure about the backend name for example, if we keep this Data~Viz~ idea we could rename it to "data" instead of "none".